docs/ref/middleware.txt

```diff
@@ -502,26 +502,41 @@ every incoming ``HttpRequest`` object. See :ref:`Authentication in web requests
 
 .. versionadded:: 5.1
 
-Redirects all unauthenticated requests to a login page. For admin views, this
-redirects to the admin login. For all other views, this will redirect to
-:setting:`settings.LOGIN_URL <LOGIN_URL>`. This can be customized by using the
+Redirects all unauthenticated requests to a login page, except for views
+excluded with :func:`~.django.contrib.auth.decorators.login_not_required`. The
+login page defaults to :setting:`settings.LOGIN_URL <LOGIN_URL>`, but can be
+customized.
+
+Enable this middleware by adding it to the :setting:`MIDDLEWARE` setting
+**after** :class:`~django.contrib.auth.middleware.AuthenticationMiddleware`::
+
+    MIDDLEWARE = [
+        "...",
+        "django.contrib.auth.middleware.AuthenticationMiddleware",
+        "django.contrib.auth.middleware.LoginRequiredMiddleware",
+        "...",
+    ]
```

Comment:
Added per @sarahboyce’s recommendation

Reply:
This is described further down this page in the [middleware ordering section](https://docs.djangoproject.com/en/dev/ref/middleware/#middleware-ordering)
I suggested adding it to the release notes as it's not uncommon we give an example on how to use something in the release notes and this would answer the question of "is this on by default"

Reply:
Yeah, it is covered down there, but I don’t think that’s particularly discoverable.

I think moving the minimal installation guide into the reference docs is better long-term, since it will always be there. Only users upgrading now will read the 5.1 release notes (even then, hopefully...).